topdown/builtins.go

```diff
@@ -88,38 +76,11 @@ func RegisterBuiltinFunc(name string, f BuiltinFunc) {
 	builtinFunctions[name] = builtinErrorWrapper(name, f)
 }
 
-// RegisterFunctionalBuiltin1 is deprecated use RegisterBuiltinFunc instead.
-func RegisterFunctionalBuiltin1(name string, fun FunctionalBuiltin1) {
```

Comment:
I think I'd rather keep them, but with deprecated comments that are picked up by the Golang tools.

Reply:
That works for me. I'm fine with keeping them included for now, as long as downstream users are very much aware that those interfaces are going away in the (potentially) near future, and that they should plan to update any custom builtins relying on those older interfaces.